src/core/load_balancing/rls/rls.cc

```diff
@@ -1814,13 +1864,17 @@ void RlsLb::RlsRequest::OnRlsCallCompleteLocked(grpc_error_handle error) {
       << "[rlslb " << lb_policy_.get() << "] rls_request=" << this << " "
       << key_.ToString() << ": response info: " << response.ToString();
   std::vector<ChildPolicyWrapper*> child_policies_to_finish_update;
+  std::vector<RefCountedPtr<ChildPolicyWrapper>> child_policies_to_delete;
+  OrphanablePtr<ChildPolicyHandler> child_policy_to_delete;
```

Comment:
Instead of needing both the vector above and this standalone pointer, I suggest changing `OnRlsResponseLocked()` and `StartUpdate()` to take a vector, so that we can just append to the vector in all cases.

Reply:
not sure what you mean, they are different types.

Reply:
Oh!  Thanks, I missed that -- I was looking at the variable names and wasn't paying close attention to the types.

Can we change the names here to make it clearer what the difference is?  For example, rename `child_policies_to_delete` to `child_policy_wrappers_to_delete` throughout?

Reply:
Done